src/lilycove_lady.c

```diff
@@ -18,215 +14,24 @@
-        gUnknown_085EEBA4,
-        gUnknown_085EEBAB
-};
-
-static const u16 Unknown_0860B23C[] = {
-        0x0056, 0x0053, 0x0054, 0x000e, 0x0012, 0x000f, 0x00a7, 0x0011, 0x0010, 0x0018, 0x0019, 0x001e, 0x0000
-};
-
-static const u16 Unknown_0860B256[] = {
-        0x0051, 0x006a, 0x006b, 0x00cc, 0x00c2, 0x002f, 0x0067, 0x0068, 0x0087, 0x0089, 0x008b, 0x009d, 0x00a0, 0x00a5, 0x0004, 0x0002, 0x0000
-};
-
-static const u16 Unknown_0860B278[] = {
-        0x0021, 0x000d, 0x001a, 0x001b, 0x001c, 0x00cc, 0x00ca, 0x0026, 0x0086, 0x0096, 0x0097, 0x009a, 0x00a2, 0x00a6, 0x0000
-};
-
-static const u16 Unknown_0860B296[] = {
-        0x0020, 0x004d, 0x004b, 0x004c, 0x0027, 0x0028, 0x0029, 0x002a, 0x002b, 0x006e, 0x005d, 0x006c, 0x006d, 0x006a, 0x006b, 0x00d6, 0x00bc, 0x00b3, 0x000b, 0x000c, 0x0000
-};
-
-static const u16 Unknown_0860B2C0[] = {
-        0x001f, 0x0013, 0x0014, 0x004a, 0x004e, 0x0049, 0x00a5, 0x00c8, 0x0067, 0x006f, 0x0000
-};
-
-static const u16 Unknown_0860B2D6[] = {
-        0x00b7, 0x00d3, 0x00d2, 0x00d8, 0x009e, 0x00a6, 0x00ab, 0x00aa, 0x006c, 0x006d, 0x0000
-};
-
-static const u16 *const gUnknown_0860B2EC[] = {
-        Unknown_0860B23C,
-        Unknown_0860B256,
-        Unknown_0860B278,
-        Unknown_0860B296,
-        Unknown_0860B2C0,
-        Unknown_0860B2D6
-};
-
-static const u16 sUnknown_0860B304[] = {
-        0x0b, 0x6e, 0x40, 0x6f, 0x44, 0x47
-};
-
-extern const u8 gUnknown_085EEB2B[];
-extern const u8 gUnknown_085EEB34[];
-extern const u8 gUnknown_085EEB3A[];
-extern const u8 gUnknown_085EEB41[];
-extern const u8 gUnknown_085EEB4A[];
-extern const u8 gUnknown_085EEB51[];
-extern const u8 gUnknown_085EEB5A[];
-extern const u8 gUnknown_085EEB61[];
-extern const u8 gUnknown_085EEB6A[];
-extern const u8 gUnknown_085EEB74[];
-extern const u8 gUnknown_085EADA4[];
-extern const u8 gUnknown_085EADB5[];
-extern const u8 gUnknown_085EADC4[];
-extern const u8 gUnknown_085EADD5[];
-extern const u8 gUnknown_085EADE7[];
-
-static const u8 *const gUnknown_0860B310[] = {
-        gUnknown_085EEB2B,
-        gUnknown_085EEB34,
-        gUnknown_085EEB3A,
-        gUnknown_085EEB41,
-        gUnknown_085EEB4A
-};
-static const u8 *const gUnknown_0860B324[] = {
-        gUnknown_085EEB51,
-        gUnknown_085EEB5A,
-        gUnknown_085EEB61,
-        gUnknown_085EEB6A,
-        gUnknown_085EEB74
-};
-static const u8 *const gUnknown_0860B338[] = {
-        gUnknown_085EADA4,
-        gUnknown_085EADB5,
-        gUnknown_085EADC4,
-        gUnknown_085EADD5,
-        gUnknown_085EADE7
-};
-
-static const u16 sUnknown_0860B34C[] = {
-        0x0120, 0x013b, 0x011e, 0x013d, 0x0019
-};
-
-static EWRAM_DATA struct LilycoveLadyFavour *gUnknown_0203CD64 = NULL;
-static EWRAM_DATA struct LilycoveLadyQuiz *gUnknown_0203CD68 = NULL;
-static EWRAM_DATA struct LilycoveLadyContest *gUnknown_0203CD6C = NULL;
+#include "contest.h"
+#include "strings.h"
+#include "constants/lilycove_lady.h"
+#include "data/lilycove_lady.h"
```

Comment:
Separate this from the other includes to distinguish it (as it's data and not definitions)